chain/client/src/client_actor.rs

```diff
@@ -9,8 +9,7 @@ use actix::dev::ToEnvelope;
 use actix::{Actor, Addr, Arbiter, AsyncContext, Context, Handler, Message};
 use actix_rt::ArbiterHandle;
 use borsh::BorshSerialize;
-use chrono::DateTime;
-use chrono::Duration as OldDuration;
+use chrono::{DateTime, Duration as OldDuration};
```

Comment:
Unrelated, but I would avoid an alias, and used `chrono::Duration` instead.

Reply:
Fixed, in a different PR.